lib/bap/bap.mli

```diff
@@ -621,12 +621,12 @@ module Std : sig
       (** Converts an 'a to a string *)
       type 'a printer = Format.formatter -> 'a -> unit
 
-      (** Interconversion between string and 'a type *)
-      type 'a converter = 'a parser * 'a printer
+      (** Interconversion between string and 'a type, along with a default *)
+      type 'a converter = 'a parser * 'a printer * 'a
 
       (** Create a parameter *)
```

Comment:
Shouldn't we actually add more documentation here? At least we should note what is the `docv` and how it can be referenced in the `doc`. And that if the default value is not specified, then a converter's default will be used.